changelog/change_add_ignoredmethods_and_ignoredclasses_to.md

```diff
@@ -0,0 +1 @@
+* [#8950](https://github.com/rubocop-hq/rubocop/pull/8950): Add `IgnoredMethods` and `IgnoredClasses` to `Lint/NumberConversion`. ([@dvandersluis][])
```

Comment:
```suggestion
* [#8950](https://github.com/rubocop-hq/rubocop/issues/8950): Add `IgnoredMethods` and `IgnoredClasses` to `Lint/NumberConversion`. ([@dvandersluis][])
```

Reply:
I'll fix this but FYI this is coming from the `rake changelog:*` tasks. I assume it's done like this because it's impossible to tell whether the reference is an issue or PR, but github redirects `pull/xxx` to `issues/xxx` if it's actually an issue so I assume that's why @marcandre did it this way.

Reply:
Agree it doesn't actually matter. Still, I'll change the script so it writes `issues/nnn` when there is `[Fix #nnn]` in the commit message, and  `pulls/x` if there isn't (assuming that the link will point to the PR to be created)